### ISSUE TITLE
startup showing errors

### ISSUE THREAD

**Issue by amalagaura** [loafoe/mattermost-client#5]
I am just getting started with mattermost-user and getting this in the hubot log:

```
INFO Logging in...
INFO Websocket URL: wss://ouraddress:443/api/v1/websocket
INFO Logged in as user "bot" but not connected yet.
INFO Connecting...
INFO Connected to Mattermost.
INFO Starting pinger...
DEBUG Found 3 profiles.
DEBUG Found 5 channels.
DEBUG Found 13 teams.
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
ERROR { [Error: getaddrinfo ENOTFOUND] code: 'ENOTFOUND', errno: 'ENOTFOUND', syscall: 'getaddrinfo' }
INFO ping
INFO ACK ping
```

Is this ping related? I cannot find the related logger code. My bot commands also do not seem to be working.

**Comment by amalagaura** [comment]
So I am getting messages but I am not getting responses.I am using hubot-matteruser. I added the user to a room and edit a message and I do get a confirmation. But I don't get any responses according to some scripts that I added.

```
DEBUG Received unhandled message type: post_edited
DEBUG { team_id: '55pbtzocjjfxpbk4z5nm9p9w7a', channel_id: '7oizum7m7fdbxbn7pez4smki4a',
  user_id: 'tzzxtju9b7bjd8e4hwkpji9m8c', action: 'post_edited', props: { post: '{"id":"zj3trphbmtrs7re5qdmehehbwa","create_at":1461700163548,"update_at":1461700166285,"delete_at":0,"user_id":"tzzxtju9b7bjd8e4hwkpji9m8c","channel_id":"7oizum7m7fdbxbn7pez4smki4a","root_id":"","parent_id":"","original_id":"","message":"bot what are the rules","type":"","props":{},"hashtags":"","filenames":[],"pending_post_id":""}' } }

```

So maybe the errors about it trying some network requests are not important? There is one every 10 seconds and all the pings do get ACK.

**Comment by amalagaura** [comment]
OK making some progress. Scripts are responding when I do not put "bot" before it. So the error's may be harmless.

**Comment by fuadsaud** [comment]
@amalagaura did you figure it out eventually? we just upgraded our mattermost server to version 3.3 and started getting some of these errors

**Comment by amalagaura** [comment]
I got my system working somehow. My error must have been different. I am getting those errors but it may not be mattermost-client because they are in hubot. It is just that hubot spits out everything without saying where the errors are from.

**Comment by loafoe** [comment]
@amalagaura I'm testing a fix for 3.3 internally now. It should be out within a day. Mattermost 3.3 API introduces breaking changes for mattermost-client

**Comment by loafoe** [comment]
@fuadsaud @amalagaura version `3.3.0` of mattermost-client and hubot-matteruser are now available with Mattermost 3.3 support
